3scale/porta
# THREESCALE-11439: Encrypt the CMS token

**What this PR does / why we need it**:

This encrypts the token used to enable the CMS edit mode in the developer portal. See the Jira description and comments for more info.

**Which issue(s) this PR fixes** 

[THREESCALE-11439](https://issues.redhat.com/browse/THREESCALE-11439)

**Verification steps** 

1. Go to Audiende -> Developer Portal -> Visit Portal
2. The developer portal should load correctly and the CMS toolbar should be visible
3. Copy the URL that includes the token
4. Open a private window and paste the URL
5. The dev. portal should load correctly and the toolbar should be visible
6. Wait a minute for the token to expire
7. Open a private window and paste the URL
8. The dev. portal should load correctly, but you shouldn't see the toolbar.
